bin/update_i18n

```diff
@@ -61,7 +68,7 @@ file_put_contents(__DIR__.'/../i18n.php', <<<EOD
  *
  * This file is automatically generated by `bin/update_i18n`.
  * Actual Gherkin translations live in cucumber/gherkin repo:
- * {$gherkinUrl}
+ * https://raw.githubusercontent.com/cucumber/cucumber/master/gherkin/gherkin-languages.json
```

Comment:
I suggest changing the URL to https://github.com/cucumber/cucumber/blob/master/gherkin/gherkin-languages.json instead, so that it links to the place where you can actually send the change (as that's what this comment is about).